decidim-core/lib/decidim/abilities.rb

```diff
@@ -3,5 +3,6 @@
 module Decidim
   module Abilities
     autoload :ParticipatoryProcessRoleAbility, "decidim/abilities/participatory_process_role_ability"
+    autoload :UserManagerAbility, "decidim/abilities/user_manager_ability"
```

Comment:
@beagleknight You need it because the file lives in `lib`, so it's not autoloaded by default by `Rails`. In the previous commit I moved those files under `app`, so the `autoload` was no longer needed.

The criteria I'm using is: if the file is a base ability class, not intended to be instantiated directly, leave it under `lib`. Otherwise keep in under `app`.

Hope it makes sense.